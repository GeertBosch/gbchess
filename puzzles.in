6k1/4Q3/5K2/8/8/8/8/8 w - - 0 1 # Mate in one
6k1/5ppp/8/7Q/2B5/7P/1r1q2P1/7K w - - 0 1 # Mate in two

