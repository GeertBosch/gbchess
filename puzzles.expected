e7 g7# 999 @ 1
h5 f7+ 999 @ 3

